test/fixtures/bats/single_line.bats

```diff
@@ -1,9 +1,9 @@
-@test "empty" { }
+@test "empty" {}
 
-@test "passing" { true; }
+@test "passing" {true;}
 
-@test "input redirection" { diff - <( echo hello ); } <<EOS
```

Comment:
Any reason for removing spaces here and not in `empty_name.bats`? I prefer the readability of spaced, but it's another styleguide question to which only conformity is important!

Reply:
Agree here. I'd rather the `tabs.bats` be changed to `whitespace.bats` and cover all the whitespace variants:
- no whitespace around single-line
- spaces around single-line
- tabs around single-line

- tabs before @test 
- tabs before test name
- tabs before test body
- spaces before @test 
- spaces before test name
- spaces before test body

Reply:
I've renamed the fixture to `whitespace.bats` and added a few extra cases, but didn't add extra cases for tabs vs. spaces, since the `[:space:]` character class matches both.

What _did_ happen is the new "extra whitespace around single-line test" case exposed a bug where the extra spaces after the closing quote were getting matched as part of the test case name. As a result, Bash would exit with a parse error when executing the preprocessed file, since the test name included the closing quote and its trailing leading to an unterminated quote parse error.

I've since updated `BATS_TEST_PATTERN` so that the name/description capture matches a non-`[:space:]` character last instead of first.